symengine/lib/symengine_wrapper.pyx

```diff
@@ -1458,6 +1556,14 @@ cdef class Complex(Number):
 
 class Infinity(Number):
 
+    @property
+    def is_infinite(self):
+        return True
+
+    @property
+    def is_prime(self):
```

Comment:
Most of the attributes directly follow from `SymPy`'s implementation. Though this is not needed currently, I thought it'd be helpful in having this as well (should an issue arise later on).